Ginger/GingerCoreNET/Run/GingerRunner.cs

```diff
@@ -2645,6 +2647,65 @@ private void CalculateModelParameterExpectedValue(Act act, ActReturnValue ARC)
         }
 
         public static void CalculateARCStatus(ActReturnValue ARC)
+        {
+            string Expression=string.Empty;
+            bool InvertResult=false;
+            switch (ARC.Operator)
+            {
+                
+                case eOperator.Contains:
+                    Expression ="\""+ ARC.Actual + "\".Contains(\"" + ARC.ExpectedCalculated + "\")";
```

Comment:
why we create expression for roslyn to cal and not do the calc in regular C# code base on the condition?